www/include/views/graphs/graph-periods.php

```diff
@@ -52,29 +52,25 @@
 $tpl = new Smarty();
 $tpl = initSmartyTpl($path, $tpl);
 
-function getGetPostValue($str)
-{
-    $value = null;
-    if (isset($_GET[$str]) && $_GET[$str]) {
-        $value = $_GET[$str];
-    }
-    if (isset($_POST[$str]) && $_POST[$str]) {
-        $value = $_POST[$str];
-    }
-    return urldecode($value);
+if (isset($_GET['chartId'])) {
```

Comment:
if $_GET['chartId'] is not retrieved, an undefined $chartId will pop in the logs line 59 and 96